### DISCUSSION TITLE
[Proposal] Launch a new project for sink connector of Apache Flink

### DISCUSSION THREAD

**Discussion by whhe** [oceanbase/oceanbase#1226] [Ideas]
As [obconnector-j](https://github.com/oceanbase/obconnector-j) has been open sourced, and we has [Flink CDC](https://github.com/ververica/flink-cdc-connectors) as source connector of Apache Flink, we can now launch a new project for sink connector of Apache Flink. 

We can do some performance optimization referencing [datax](https://github.com/alibaba/datax), and support the MySQL mode of OceanBase in the first release.

**Comment by longdafeng** [comment]
If just for Flink, why not contribute to Flink project. 

But if we plan to implement multiple sink connector, not only for Flink, but also for Chunjun/TapData, setting up one project make sense.

**Reply by whhe** [reply]
According to the [disscussion](https://www.mail-archive.com/user@flink.apache.org/msg44405.html) in the mail list, Flink community is going to move most of connectors outside the Flink repository, just like [flink-connector-jdbc](https://github.com/apache/flink-connector-jdbc).
And for now it is a common practice for database brands to maintain the Flink/Spark connector under their own organizations. In this way, we can have a strong control over the iteration cycle,  requirements priority and testing standard, which is very meaningful for a new project.

**Comment by whhe** [comment]
The repository is public now.  

https://github.com/oceanbase/flink-connector-oceanbase
